### PR TITLE
Use github actions

### PR DESCRIPTION
**Pull request by kelloggm**

### CODE REVIEW

**Comment by kelloggm** [comment]
This looks like it's working, but I figured another set of eyes would be helpful @msridhar

**Review by msridhar — APPROVED** [review]
👍

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Java CI
+
+on: [push]
```
Comment (mernst) [inline .github/workflows/gradle.yml:3]: This should include pull-request as well.

Comment (kelloggm) [reply .github/workflows/gradle.yml]: Good idea, thanks

**Pull request merged**